Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -642,9 +642,8 @@ def doNotInlineEventContent(instance,label = "cms.untracked.vstring(process."+th
                         output.dataset.filterName = cms.untracked.string('StreamALCACombined')
 
                 if "MINIAOD" in streamType:
-                    output.dropMetaData = cms.untracked.string('ALL')
-                    output.fastCloning= cms.untracked.bool(False)
-                    output.overrideInputFileSplitLevels = cms.untracked.bool(True)                      
+                    from PhysicsTools.PatAlgos.slimming.miniAOD_tools import miniAOD_customizeOutput
+		    miniAOD_customizeOutput(output)
```

Comment:
it is interesting that the python intepreter understands things correctly while github doesn't